torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -2289,6 +2325,14 @@ RegisterOperators reg2({
           "aten::reverse(" decl_type "[](a!) self) -> ()",                     \
           listReverse<value_type>,                                             \
           aliasAnalysisFromSchema()),                                          \
+      Operator(                                                                \
```

Comment:
can we make these in the `aten` namespace instead of `prim` since these two have schema.

Reply:
hmmm seems like the previous one already in `prim`.. nvm.